satpy/readers/fci_l1c_nc.py

```diff
@@ -134,6 +143,13 @@
 }
 
 
+def _get_boolean_env_variable(variable, default=True):
+    var = os.getenv(variable)
+    if var is None:
+        return default
+    return var.lower() in ('true', '1', 'yes')
```

Comment:
So Satpy has already a bunch of config tools (https://github.com/pytroll/satpy/blob/main/satpy/_config.py) and is using donfig, which means that adding there would automatically give the user multiple ways of configuring this.

But now that I think of it, this setting is reader-specific as it is only use here. Should it be a reader kwarg instead?